locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
> No, because nvm_version_path does different things depending on the version number.

After running `declare -f nvm_version_path` I see what you mean. Thanks

So another attempt:
```shell
$ env VERSION=`python tools/getnodeversion.py` make install DESTDIR=`nvm_version_path $VERSION` PREFIX=""
```
Perhaps not pretty but might an acceptable solution until support for this lands in nvm like you mentioned before?